charts/portworx/v1.0.9/questions.yml

```diff
@@ -0,0 +1,921 @@
+  options:
+    - "standard"
+    - "ssd"
+
+- variable: drive_10.gc.size
+  show_if: "addGCDrive_10=true"
+  description: "Volume size"
+  label: "Size"
+  type: int
+  required: true
+  group: "Storage Parameters"
+
+- variable: maxStorageNodes
+  show_if: "environment=Cloud&&deviceConfig=Create Using a Spec"
+  description: "Max storage nodes per availability zone"
+  label: "Max storage nodes (Optional)"
+  type: int
+  group: "Storage Parameters"
+
+################################### Network options ################################
+- variable: dataInterface
+  description: "Specify your data network interface (example: `eth1`). If set to `auto`, Portworx will automatically select the first routable interface."
+  type: string
+  label: "Data Network Interface"
+  default: auto
+  group: "Network Parameters"
+- variable: managementInterface
+  description: "Specify your management network interface (example: `eth1`). If set to `auto`, Portworx will automatically select the first routable interface."
+  type: string
+  default: auto
+  label: "Management Network Interface"
+  group: "Network Parameters"
+
+################################### Platform options ################################
+- variable: platformOptions
+  type: enum
+  label: "Platform"
+  group: "Platform Parameters"
+  options:
+    - "AKS"
+    - "EKS"
+    - "GKE"
+
+################################### Registry settings options ################################
+- variable: customRegistry
+  label: "Use a custom container registry?"
+  type: boolean
+  default: false
+  group: "Container Registry Parameters"
+
+- variable: registrySecret
+  show_if: "customRegistry=true"
+  description: "Specify a custom Kubernetes secret that will be used to authenticate with a container registry. Must be defined in kube-system namespace. (example: regcred)"
+  type: string
+  label: "Registry Kubernetes Secret"
+  group: "Container Registry Parameters"
+- variable: customRegistryURL
+  show_if: "customRegistry=true"
+  description: "Specify a custom container registry server (including repository) that will be used instead of index.docker.io to download Docker images. (example: dockerhub.acme.net:5443 or myregistry.com/myrepository/)"
+  label: "Custom Registry URL"
+  type: string
+  group: "Container Registry Parameters"
+
+
+
+################################## Optional features ############################
+# TODO: Once we have a stable CSI release, we will default this to enabled
+#- variable: csi
+#  description: "Select if you want to enable CSI (Container Storage Interface). CSI is still in ALPHA."
+#  type: boolean
+#  label: "Enable CSI"
+#  default: false
+#  required: false
+#  group: "Advanced parameters"
+
+- variable: storkVersion
+  default: "2.3.1"
+  label: "Stork version"
+  type: string
+  group: "Advanced parameters"
+
+- variable: lighthouseVersion
+  default: "2.0.5"
+  type: string
+  label: "Lighthouse version"
+  group: "Advanced parameters"
```

Comment:
These are outdated. Stork is 2.4.0 and Lighthouse is 2.0.7

Reply:
done